examples/jf-control-app/commands/pairing/PairingCommand.cpp

```diff
@@ -577,17 +645,20 @@ void PairingCommand::OnCommissioningComplete(NodeId nodeId, const Optional<Crypt
                 }
             }
 
-            auto call = rpcClient.TransferOwnership(request, OnOwnershipTransferDone);
+            auto call = rpcClient.TransferOwnership(request, OnRPCTransferDone);
             if (!call.active())
             {
                 ChipLogError(JointFabric, "RPC: OwnershipTransfer Call Error");
-                // The RPC call was not sent. This could occur due to, for example, an invalid channel ID. Handle if necessary.
+                SetCommandExitStatus(CHIP_ERROR_SHUT_DOWN);
```

Comment:
same comment on shut down

Reply:
probably the clean solution would be to do a 1:1 mapping between RPC error codes and Matter stack error codes. Will be handled in https://github.com/project-chip/connectedhomeip/issues/39876